src/dsp_tools/commands/excel2json/models/input_error_lists.py

```diff
@@ -40,6 +42,22 @@ def execute_error_protocol(self) -> str:
         raise NotImplementedError
 
 
+@dataclass
+class CollectedSheetProblems:
```

Comment:
The serialisation result of this function is used in a unittest. It did not change the result to the previous serialisation